src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -107,12 +111,26 @@ function OptionRowLHN(props) {
             ReportUtils.isWaitingForIOUActionFromCurrentUser(optionItem) ||
             (optionItem.isTaskReport && optionItem.isTaskAssignee && !optionItem.isCompletedTaskReport && !optionItem.isArchivedRoom));
 
+    useFocusEffect(
+        useCallback(() => {
+            isFocusedRef.current = true;
+            return () => {
+                if (!isSmallScreenWidth) {
+                    return;
+                }
+                isFocusedRef.current = false;
+            };
+        }, []),
+    );
     /**
      * Show the ReportActionContextMenu modal popover.
      *
      * @param {Object} [event] - A press event.
      */
     const showPopover = (event) => {
+        if (!isFocusedRef.current) {
```

Comment:
```suggestion
        if (!isFocusedRef.current && isSmallScreenWidth) {
```

@redpanda-bit We can't use the memoized value returned by the useCallBack as `isSmallScreenWidth` is subject to change. The other way around might be to pass `isSmallScreenWidth` as dependency in the related useCallBack.

Reply:
I totally missed that. Thanks for catching that. Made the changes and re-tested. Please let me know if you see something else.